<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="1,1,1">
   <item row="4" column="2" rowspan="2">
    <widget class="QPushButton" name="pbConnect">
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="cbSource">
     <item>
      <property name="text">
       <string>rtsp</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>usb</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>usb 12bit</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Source:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>ip (sender)</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="pbStop">
     <property name="text">
      <string>Stream</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>port (sender)</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>806</width>
        <height>427</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="lbVideoArea">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Width (reader)</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="leIp">
     <property name="font">
      <font>
       <family>Ubuntu Mono</family>
      </font>
     </property>
     <property name="inputMask">
      <string>000.000.000.000;_</string>
     </property>
     <property name="text">
      <string>192.168.1.2</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QSpinBox" name="sbPort">
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>9999999</number>
     </property>
     <property name="value">
      <number>7777</number>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Height (reader)</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QSpinBox" name="sbReaderWidth">
     <property name="maximum">
      <number>99999</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>1920</number>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QSpinBox" name="sbReaderHeight">
     <property name="maximum">
      <number>99999</number>
     </property>
     <property name="value">
      <number>1080</number>
     </property>
    </widget>
   </item>
   <item row="6" column="2" rowspan="2">
    <widget class="QPushButton" name="pbSetUpReader">
     <property name="text">
      <string>Set up reader</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
